device/nokia/armhf-nokia_ixs7215_52x-r0/installer.conf

```diff
@@ -1 +1,2 @@
 VAR_LOG_SIZE=4096
+extra_cmdline_linux="logs_inram=on"
```

Comment:
I am worries on this platform, it doesn't have too much mem either. Do you have a standard to judge if we need loginram for a platform?

Reply:
@Blueve 7215 has around 3GB of total memory. So /var/log size on tmpfs will be max 10% of total memory which comes to around 300MB. Currently it has 2.2GB free:-

```
root@str-2-7215-acs-4:~# free -h
              total        used        free      shared  buff/cache   available
Mem:          3.0Gi       580Mi       1.9Gi        57Mi       500Mi       2.2Gi
Swap:            0B          0B          0B
root@str-2-7215-acs-4:~# 
```

Reply:
But var log size was set to 4GB... you can refer to LN1: `VAR_LOG_SIZE=4096`

Reply:
@Blueve checked with Guohan...seems fine. We need to live with this limitation of less memory but cannot use disk for too much writes

Reply:
@Blueve , what rotation issue?